plugin.xml

```diff
@@ -54,6 +54,8 @@
         <source-file src="src/android/FileTransfer.java" target-dir="src/org/apache/cordova/filetransfer" />
         <source-file src="src/android/FileProgressResult.java" target-dir="src/org/apache/cordova/filetransfer" />
         <source-file src="src/android/FileUploadResult.java" target-dir="src/org/apache/cordova/filetransfer" />
+
+        <framework src="com.google.android.gms:play-services-base:15.0.0+" />
```

Comment:
@davidofwatkins can you make this line:

```
 <preference name="PLAY_SERVICES_VERSION" default="15.0.0"/>
 <framework src="com.google.android.gms:play-services-base:$PLAY_SERVICES_VERSION" />
```

and also add in an engine tag:

```
<engines>
<engine name="cordova" version=">=7.1.0"/>
<engine name="cordova-android" version=">=6.3.0"/>
</engines>
```

This gets around the Play Services issue: https://blog.phonegap.com/the-play-services-problem-with-cordova-applications-489c9aeb2e89

Reply:
Ah, I wasn't aware of this. Good to know, thanks!